Error generating javadoc with JDK 1.8

[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 22.037 s
[INFO] Finished at: 2018-09-14T23:27:26+02:00
[INFO] Final Memory: 29M/436M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.0.1:jar (default-cli) on project openkm: MavenReportException: Error while generating Javadoc:
[ERROR] Exit code: 1 - /home/jllort/git/document-management-system/src/main/java/com/openkm/kea/RDFREpository.java:62: warning: no @return
[ERROR] public RepositoryConnection getSKOSConnection() throws RepositoryException {

--- a/javadoc.sh
+++ b/javadoc.sh
@@ -1,0 +1,2 @@
+#!/bin/bash
+./mvnw javadoc:javadoc -Dmaven.javadoc.failOnError=false

--- a/pom.xml
+++ b/pom.xml
@@ -182,7 +182,7 @@
     <dependency>
       <groupId>commons-fileupload</groupId>
       <artifactId>commons-fileupload</artifactId>
-      <version>1.3.1</version>
+      <version>1.3.2</version>
     </dependency>
     <dependency>
       <groupId>commons-httpclient</groupId>

